crypto/tls/tls.go

```diff
@@ -72,10 +72,17 @@ func (cfg *ClientConfig) GetTLSConfig() (*tls.Config, error) {
 	return config, nil
 }
 
-// GetGRPCDialOptions creates GRPC DialOptions for TLS
+// WithInsecure wraps grpc.WithInsecure.
+//
+// Stubbable for tests.
+var WithInsecure = func() grpc.DialOption {
+	return grpc.WithInsecure()
+}
+
+// GetGRPCDialOptions creates GRPC DialOptions for TLS.
 func (cfg *ClientConfig) GetGRPCDialOptions(enabled bool) ([]grpc.DialOption, error) {
```

Comment:
[nit] Can we rename enabled to tlsEnabled to make it more clear?

Reply:
@pracucci done.